src/sugar3/graphics/radiopalette.py

```diff
@@ -62,6 +80,9 @@ def __init__(self, **kwargs):
         self.set_content(self.button_box)
 
     def append(self, button, label):
+        '''
+        Adds a widget to the RadioPalette
```

Comment:
Please document the parameters, as per the example patch included in the gci task.

Also please document how this function ".show()s" the button, deals with it being clicked (it does right?) and says the button must not have subpalettes.  The activity developer should understand what this does using only your peices of text (and not seeing the code).